### PR TITLE
Support es5 by default

### LINKED ISSUE #4: Improve the build output
We currently support two ways of importing a component:

1.  `import { ErrorPanel } from '@mozilla-frontend-infra/components`
2. `import ErrorPanel from '@mozilla-frontend-infra/components/build/ErrorPanel`

An improved way would be to have `2.` not have to go through the `build/` directory.

```js
import ErrorPanel from '@mozilla-frontend-infra/components/ErrorPanel';
```

Discussion: https://github.com/mozilla-frontend-infra/components/pull/1#discussion_r201191458

### PR DESCRIPTION
**Pull request by helfi92**
* Copy other files to the build directory
* Make the publish script only publish the build directory. This way we can have cleaner imports.

Resolves #4.

### CODE REVIEW

**Comment by helfi92** [comment]
@eliperelman `yarn build` creates a `build` and `es5` output directory. Now that this PR targets older browsers, I don't think we need the `es5` folder. What's the best way to remove it using `.neutrinorc.js`?

File: package.json [hunk 1]
```diff
@@ -4,23 +4,19 @@
   "description": "Mozilla Front End Infra Shareable React Components",
   "repository": "mozilla-frontend-infra/components",
   "license": "MPL-2.0",
-  "main": "build/index.js",
+  "main": "./index.js",
```
Comment (eliperelman) [inline package.json]: Doesn't this still build to the `build` directory?

File: package.json [hunk 2]
```diff
@@ -4,23 +4,19 @@
...
   "scripts": {
     "changelog": "auto-changelog",
     "start": "neutrino styleguide:start",
     "build": "neutrino build",
-    "prepare": "neutrino build",
     "deploy": "neutrino styleguide:build && gh-pages --remote upstream -d styleguide",
     "lint": "neutrino lint",
+    "publish": "yarrn build && npm publish build",
```
Comment (eliperelman) [inline package.json]: s/yarrn/yarn/?

File: package.json (hunk 1 above)
Comment (helfi92) [reply package.json]: Yes, but the `package.json` will be copied to the `build` directory. Inside `.neutrinorc.js` I added:

```js
neutrino.on('build', () => {
  ['package.json', 'logo.png', 'LICENSE', 'README.md', 'AUTHORS'].map(file => {
    fs.copyFileSync(file, join(__dirname, `build/${file}`));
  })
});
```

**Comment by eliperelman** [comment]
Right now there isn't an easy way to disable that, but I had started working on a PR to the preset to fix that. In Neutrino v9 it won't be a problem at all. Worst case you may be able to re-build the preset manually by pulling in the distinct parts from the package.

**Comment by helfi92** [comment]
Sounds good. Thanks for the review 🙌. This is finally ready to be published :)

**Pull request merged**